modal/cls.py

```diff
@@ -75,6 +75,7 @@ def _get_class_constructor_signature(user_cls: type) -> inspect.Signature:
 
 @dataclasses.dataclass()
 class _ServiceOptions:
+    # Note that default values should always be "untruthy" so we can detect when they are not set
```

Comment:
Can we add a test to ensure this is always the case?

Reply:
👍 good idea!